.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "moose"]
 	path = moose
-	url = https://github.com/idaholab/moose.git
```

Comment:
This change should be reverted before CI testing begins, otherwise there will likely be failures.

Reply:
Hello, thank you for the quick response. The MOOSE PR hasn't been merged yet, so the changes are still just on my fork of MOOSE. This PR was to test whether the changes here will be forward compatible. Is there another way to test this? Thank you.

Reply:
This is ok for testing purposes, but normally we would stage this fix PR without a submodule change until the upstream PR merged into MOOSE. Then we update the submodule hash to the new "next" branch, and then do final testing and merge. This ensures that we're using an up-to-date hash of MOOSE that will always be available.

Reply:
Is this good for now? Do I come back once the PR is merged into MOOSE?

Reply:
This has been addressed.